test/runtests.jl

```diff
@@ -34,19 +34,6 @@ end
         @test isdir("Git.jl")
         @test isfile(joinpath("Git.jl", "Project.toml"))
     end
```

Comment:
Out of curiosity, is there a reason for doing the same block of tests twice?

Reply:
IIRC, we want to call the `git()' function multiple times as part of our "make sure the package is thread-safe" tests? @giordano

Reply:
That looks suspiciously sequential for testing thread-safety.

Reply:
No, I suspect that's an artifact of refactoring the tests.

Reply:
We should add a test that spawns multiple tasks and makes sure that the clones all work normally. And then at the end, checks that still no environment variables have leaked.